Documentation/bootstrapping-guidelines.md

```diff
@@ -0,0 +1,96 @@
+
+To build .NET SDK from source requires previously source-built .NET SDK and toolset which is also referred as bootstrapping .NET SDK and toolset. Check quick [instructions](https://github.com/dotnet/installer/blob/main/README.md#build-net-from-source-source-build) to build SDK using source-build. This document provides guidance for getting or building  bootstrapping .NET SDK and toolset in different scenarios (covering step 2 and step 3 of source-build instructions).
+
```

Comment:
Before diving into details, I would add a high-level description of the steps the user will perform (and the reasons why, if necessary).